docs/runkit_example.js

```diff
@@ -1,13 +1,17 @@
-const { registerHtml } = require('tram-one')
+// NOTE: This won't work since Tram-One does not (right now) support server-side rendering
+// This is on our backlog though, and if it is something that you would like, please vote / comment here:
+// https://github.com/Tram-One/tram-one/issues/181
```

Comment:
Domino was removed, so this runkit example won't work, but I'm keeping it here because one day it hopefully will :smile: